Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM maven:3.5-jdk-8
+FROM openjdk:8u151-jdk-alpine
```

Comment:
I heard there were some issues with `snappy` on alpine before. FYI `snappy` can be configured to turn on compression for kafka producer.

Reply:
we can keep the maven image and do a cleanup after instead
it saves about 300MB